dask/dataframe/core.py

```diff
@@ -484,22 +483,34 @@ def map_overlap(self, func, before, after, *args, **kwargs):
         ...                    'y': [1., 2., 3., 4., 5.]})
         >>> ddf = dd.from_pandas(df, npartitions=2)
 
-        The pandas ``diff`` method computes a discrete difference shifted by a
-        number of periods (can be positive or negative).  This can be
-        implemented by mapping calls to ``df.diff`` to each partition after
-        prepending/appending that many rows, depending on sign:
+        A rolling sum with a trailing moving window of size 2 can be computed by
+        overlapping 2 rows before each partition, and then mapping calls to
+        ``df.rolling(2).sum()``:
```

Comment:
Ah, I see now how this is more complex.  I was thinking of mapping the `sum` function across, but I see now how this is was a bit naive.